server/models.py

```diff
@@ -993,6 +994,11 @@ def files(self):
         else:
             return {}
 
+    def external_files_dict(self):
+        """ Return a dictionary of filenames to contents."""
```

Comment:
I don't think this docstring is accurate, it has ExternalFile objects not contents